boards/infineon/kit_xmc72_evk/kit_xmc72_evk_xmc7200d_e272k8384_m0p.dts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2025 Cypress Semiconductor Corporation.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <arm/infineon/cat1c/mpns/xmc7200_e272k8384.dtsi>
+#include <arm/infineon/cat1c/xmc7200/system_clocks.dtsi>
+#include <arm/infineon/cat1c/xmc7200/xmc7200_m0p.dtsi>
+#include <arm/infineon/cat1c/xmc7200/memory_partition.dtsi>
+#include "kit_xmc72_evk_xmc7200d_e272k8384_m0p-pinctrl.dtsi"
+#include "kit_xmc72_evk_common.dtsi"
+
+/ {
+	model = "kit_xmc72_evk_xmc7200d_e272k8384 with an Cypress PSOC™ 6 SoC";
```

Comment:
The comment about PSOC 6 SoC doesn't make sense. Please clarify the meaning here.